tools/ovc/unit_tests/ovc/utils/cli_parser_test.py

```diff
@@ -343,12 +343,81 @@ def test_mo_convert_params(self):
         assert argv.input_model == args['input_model']
         assert argv.extension == args['extension']
         assert argv.input == ['name', ("a", [1,2,3], ov.Type.f32)]
-        assert argv.output == ["a","b","c"]
+        assert argv.output == ["a", "b", "c"]
 
         for arg, value in vars(argv).items():
             if arg not in args and arg != 'is_python_api_used':
                 assert value == cli_parser.get_default(arg)
 
+    def test_output_post_parsing_1(self):
+        args = {'input_model': os.path.dirname(__file__),
+                'input': "input_1[1,2,3]",
+                'output_model': os.getcwd() + "model.xml",
+                "framework": "onnx",
+                'verbose': False,
+                'output': "a,b,c"}
+
+        argv = args_to_argv(**args)
+
+        argv.is_python_api_used = False
+        argv = arguments_post_parsing(argv)
```

Comment:
I don't think that just tests for one internal function is enough. Could we have tests that check outside visible behavior with clear separation between CLI and openvino.compile_model.

Reply:
Added a couple of more solid test cases.

One solely for `convert_model` for a model with output name `x,y`. (cli tool can not process such outputs)
And one case where 2 outputs separated b coma (for cli) or specified in a list (for convert_model).

https://github.com/openvinotoolkit/openvino/pull/19391/files#diff-e3878abd42ca567a18e9b90e4bd37a1394a1db1106502b0ee15b4985fc2fb856R263-R279